Hazel/src/Hazel/ImGui/ImGuiLayer.h

```diff
@@ -6,12 +6,14 @@
 #include "Hazel/Events/KeyEvent.h"
 #include "Hazel/Events/MouseEvent.h"
 
+#include "ImGuiConsole.h"
+
 namespace Hazel {
 
 	class HAZEL_API ImGuiLayer : public Layer
 	{
 	public:
-		ImGuiLayer();
+		ImGuiLayer(bool renderDemo = true, bool renderConsole = true);
```

Comment:
One thing to be careful with adding a bunch of boolean variables like this is that if someone adds another one or reorders them then they will be the incorrect boolean combination with no warnings or errors